src/Microsoft.DotNet.Wpf/src/WpfGfx/codegen/mcg/tools/InvokeCSP.cmd

```diff
@@ -11,7 +11,7 @@ if "%HostArch%"=="x86" set HostArch=i386
 set HostArch=i386
 
 set CspSrcPath=%_NtBinDir%\wpf\src\Graphics\tools\csp
-set CspExePath=%_NTTREE%\wpf
+if "%CspExePath%" == "" set CspExePath=%_NTTREE%\wpf
```

Comment:
I think we should throw here, _NTTREE used to be the path of the location where the binaries are produced.

Reply:
I removed the if instead, it was redundant since later in the script we check that csp.exe exists in '%CspExePath%'.